test/LocatedTriplesTest.cpp

```diff
@@ -0,0 +1,704 @@
+// Copyright 2023 - 2024, University of Freiburg
+//  Chair of Algorithms and Data Structures.
+//  Authors:
+//    2023 Hannah Bast <bast@cs.uni-freiburg.de>
+//    2024 Julian Mundhahs <mundhahj@tf.uni-freiburg.de>
+
+#include <gtest/gtest.h>
+
+#include "./util/AllocatorTestHelpers.h"
+#include "./util/IdTableHelpers.h"
+#include "./util/IdTestHelpers.h"
+#include "index/CompressedRelation.h"
+#include "index/IndexImpl.h"
+#include "index/LocatedTriples.h"
+#include "index/Permutation.h"
+
+// TODO: Why the namespace here? (copied from `test/IndexMetaDataTest.cpp`)
```

Comment:
This is an anonymous namespace (similar but more powerful to `static`.
The things in it will not be linked to other translation units and therefore not lead to multiple definition errors.
Something like this should always be done for symbols trhat only appear in a .cpp file.

Reply:
maybe extend it to your Matchers below :)

Reply:
Good to know :) That was a remnant from Hannah's original code.

Reply:
I suspect that more and more powerful Matchers will be needed. Something like `LocatedTriplesTestHelpers.h` might be a good solution then..